### PR TITLE
send pull request updated event on participant approved event

### PR DESCRIPTION
**Pull request by shmish111**

### CODE REVIEW

File: src/main/java/nl/topicus/bitbucket/api/PullRequestListener.java [hunk 1]
```diff
@@ -17,10 +12,7 @@
 import com.atlassian.httpclient.api.Request;
 import com.atlassian.httpclient.api.Response;
 import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
-import nl.topicus.bitbucket.events.BitbucketPushEvent;
-import nl.topicus.bitbucket.events.BitbucketServerPullRequestEvent;
-import nl.topicus.bitbucket.events.EventType;
-import nl.topicus.bitbucket.events.Events;
+import nl.topicus.bitbucket.events.*;
```
Comment (Eernie) [inline src/main/java/nl/topicus/bitbucket/api/PullRequestListener.java]: Could you please revert this?

File: src/main/java/nl/topicus/bitbucket/api/PullRequestListener.java [hunk 2]
```diff
@@ -1,11 +1,6 @@
 package nl.topicus.bitbucket.api;
 
-import com.atlassian.bitbucket.event.pull.PullRequestEvent;
-import com.atlassian.bitbucket.event.pull.PullRequestMergedEvent;
-import com.atlassian.bitbucket.event.pull.PullRequestOpenedEvent;
-import com.atlassian.bitbucket.event.pull.PullRequestReopenedEvent;
-import com.atlassian.bitbucket.event.pull.PullRequestRescopedEvent;
-import com.atlassian.bitbucket.event.pull.PullRequestUpdatedEvent;
+import com.atlassian.bitbucket.event.pull.*;
```
Comment (Eernie) [inline src/main/java/nl/topicus/bitbucket/api/PullRequestListener.java]: Could you please revert this?

**Comment by shmish111** [comment]
@Eernie I have started to have a look at disabling the new event that I have added however I only think it makes sense to do this if you do it for all events and add all events (that's quite a lot). Also, architecturally, it makes much more sense to send everything and have the consumer filter events. Are you sure you want to go down this path? What would you like me to do?

**Comment by Eernie** [comment]
@shmish111 I'll have to implement the possibility to turn off/on different events. Just like the cloud version. I hope I'll get to it somewere this week.

**Comment by shmish111** [comment]
Hi @Eernie I wonder if you got anywhere with this? Is there anything I can do?

**Comment by Eernie** [comment]
Yes i got the basis for it. But I still need to finish it.

**Comment by Macarse** [comment]
@Eernie What's the status of your implementation? If I build https://github.com/topicusfinan/bitbucket-webhooks-plugin/issues/35 I would require the turn on/off different events.

**Comment by Eernie** [comment]
I'm almost there #40

**Comment by shmish111** [comment]
What's the news on this these days?

**Comment by casz** [comment]
@shmish111 I have reworked this PR inside #64

**Comment by shmish111** [comment]
great @casz I'll close this

**Pull request closed without merging**